packages/object-lists/migrations/0000_bored_silver_surfer.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `object_list_item` (
+	`id` serial AUTO_INCREMENT NOT NULL,
+	`object_id` varchar(256) NOT NULL,
+	`list_id` int,
+	`created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	`created_by` varchar(256) NOT NULL,
+	CONSTRAINT `object_list_item_id` PRIMARY KEY(`id`),
+	CONSTRAINT `object_list_item_object_id_list_id_unique` UNIQUE(`object_id`,`list_id`)
+);
+--> statement-breakpoint
+CREATE TABLE `object_list` (
```

Comment:
For consideration: if the migration of the first `CREATE TABLE` succeeds, but this one fails, what will be the state of the database? Can we easily re-apply the failed migration? Or will it fails because the first table has been created?